### DISCUSSION TITLE
cassete tape on ibm 5150 does not work

### DISCUSSION THREAD

**Discussion by doutter** [86Box/86Box#4955] [General]
I am using build 6130

Using 8088 (IBM PC 1981 or 1982)

I am using instructions in this video [https://www.youtube.com/watch?v=G_8CXyF5M1Q](url) and it is not working.

I remember to use that in the past (I don't remember the build) and it works.

**Comment by jriwanek** [comment]
Could you provide the cassette image you're using?
